grails-app/utils/org/pih/warehouse/DateFormatName.groovy

```diff
@@ -0,0 +1,21 @@
+package org.pih.warehouse
+
+// Enum for storing labels used in dates formatting.
+// In case of adding new format we have to:
+// 1. Add new format in message.properties file
+// 2. Add the newly created label to this enum
+// This enum is used in:
+// 1. G tag - <g:formatDate date={your date} formatName={enum property} />
+// 2. LocalizationUtil.formatDate(your date, enum property)
+
+enum DateFormatName {
+    DATE_TIME('default.date.format'),
+    FULL_DATE('custom.date.format'),
+    MONTH_YEAR('expiry.date.format'),
```

Comment:
thanks for making these changes but now the actual properties themselves don't match.

I was thinking we'd have something like:
```
datetime.format=DD/MMM/yyyy hh:mm:ss a z
date.format=DD/MMM/yyyy
monthyear.format=MMM/yyyy
```
the monthyear one is hard to decide on what a good wording would be but maybe it's clear enough like that.